### DISCUSSION TITLE
multi domain certificate

### DISCUSSION THREAD

**Discussion by Prunkton** [polymind-inc/acmebot#219] [Q&A] [answered]
I would like to issue a multi domain certificate for e.g.
website1.com
website1.fr
website2.com
etc...

Since I have different domains, I have to create an Azure DNS Zone for every single domain. While following the workflow from the README.md I come the the point I can submit my domains via https://YOUR-FUNCTIONS.azurewebsites.net/add-certificate. At this point I'm not able to add domains from different DNS Zones, because my already added domains will get remove as soon I choose another DNS Zone.
Is this the right workflow or do I miss something here?

**Comment by shibayan** [comment, marked as answer]
It's the right behavior. Currently, certificates can only be issued for a single domain.

I will add it to the issue to see if it can be supported. Thank you.

**Reply by Prunkton** [reply]
thanks for the quick response :)

**Reply by shibayan** [reply]
There was no problem issuing multi-domain certificates with the existing implementation. It will be available in the next version.

**Reply by Prunkton** [reply]
its working as expected, thanks a lot!
